src/compiler/scala/tools/nsc/javac/JavaParsers.scala

```diff
@@ -419,10 +468,10 @@ trait JavaParsers extends ast.parser.ParsersCommon with JavaScanners {
 
     def typeParam(): TypeDef =
       atPos(in.currentPos) {
-        annotations()
+        val anns = annotations()
         val name = identForType()
         val hi = if (in.token == EXTENDS) { in.nextToken() ; bound() } else EmptyTree
-        TypeDef(Modifiers(Flags.JAVA | Flags.DEFERRED | Flags.PARAM), name, Nil, TypeBoundsTree(EmptyTree, hi))
+        TypeDef(Modifiers(Flags.JAVA | Flags.DEFERRED | Flags.PARAM, typeNames.EMPTY, anns), name, Nil, TypeBoundsTree(EmptyTree, hi))
```

Comment:
what's wrong with `tpnme`?

Reply:
In scala.reflect.internal cake, nothing at all.  In the scala.reflect.api cake, it's ❌ deprecated.  Btw, this is your code. 😛

Reply:
!